thirdparty/kyaml/runfn/runfn.go

```diff
@@ -110,6 +108,21 @@ func (r RunFns) Execute() error {
 	return r.runFunctions(nodes, output, fltrs)
 }
 
+func (r RunFns) functionConfigFilterFunc() (kio.LocalPackageSkipFileFunc, error) {
```

Comment:
Please add a godoc explaining the high level filter logic.

Reply:
added